uaclient/status.py

```diff
@@ -394,9 +394,9 @@ def _get_config_status(cfg) -> Dict[str, Any]:
         "features": cfg.features,
     }
     # LP: #2004280 maintain backwards compatibility
-    ua_config = {}
+    ua_config = user_config_file.user_config.public_config.to_dict()
     for key in UA_CONFIGURABLE_KEYS:
-        if hasattr(cfg, key):
+        if hasattr(cfg, key) and ua_config[key] is None:
             ua_config[key] = getattr(cfg, key)
     ret["config"]["ua_config"] = ua_config
```

Comment:
@orndorffgrant  Going through the extra steps here since the default config has all values set to `None`, and these values are shown in the status-cache as None as well instead of the default values. The default values are returned when `getattr()` is called on the `UAConfig` class. This sets the correct default values for keys like `apt_news_url`, `metering_timer`.

Reply:
Thanks sorry I missed that! This is good as is